openmdao/core/driver.py

```diff
@@ -680,11 +680,11 @@ def _update_voi_meta(self, model):
             else:
                 objs[name] = data
 
-        response_size = np.sum(resps[n]['size'] for n in self._get_ordered_nl_responses())
+        response_size = sum(resps[n]['size'] for n in self._get_ordered_nl_responses())
```

Comment:
I doubt that it would be faster to construct an ndarray and then call np.sum on it, but in any case this isn't something that will have any noticeable effect on performance either way.